site/content/en/latest/user/observability/rate-limit-observability.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "RateLimit Observability"
+---
+
+Envoy Gateway provides observability for the RateLimit instances.
+This guide show you how to config RateLimit observability, includes traces.
+
+## Prerequisites
+
+Follow the steps from the [Quickstart Guide](../../quickstart) to install Envoy Gateway and the HTTPRoute example manifest.
+Before proceeding, you should be able to query the example backend using HTTP. Follow the steps from the [Global Rate Limit](../../traffic/global-rate-limit) to install RateLimit.
```

Comment:
```suggestion
Before proceeding, you should be able to query the example backend using HTTP. Follow the steps from the [Global Rate Limit](../traffic/global-rate-limit) to install RateLimit.
```

Reply:
I'm not very familiar with hugo, I just found out about it today, thanks